packages/react/src/themes/teams/components/Toolbar/toolbarVariables.ts

```diff
@@ -0,0 +1,86 @@
+import { pxToRem, stringLiteralsArray } from '../../../../lib'
+import { extendColorScheme, pickValuesFromColorScheme } from '../../../colorUtils'
+import { ItemType } from '../../../types'
+import { TeamsSchemeMappingWithAreas } from '../../types'
+
+export const toolbarColorAreas = stringLiteralsArray(
+  'foreground1',
+  'border',
+
+  'foregroundActive',
+
+  'foregroundHover',
+  'backgroundHover',
+
+  'foregroundFocus',
+  'backgroundFocus',
+  'borderFocus',
+
+  'foregroundDisabled1',
```

Comment:
nit: we may want to extend the color scheme, and use the `foregroundDisabled1` for the `foregroundDisabled`, so it would be more user friendly, but then it may raise a problem because the user will think this one is acttually the `foregroundDisabled` from the color scheme... Let's just think about this and decide together whether we want that.